module/epochs/qc_voter.go

```diff
@@ -34,16 +44,16 @@ func NewRootQCVoter(
 	me module.Local,
 	signer hotstuff.Signer,
 	state protocol.State,
-	client module.QCContractClient,
+	contractClients []module.QCContractClient,
 ) *RootQCVoter {
 
 	voter := &RootQCVoter{
-		log:    log.With().Str("module", "root_qc_voter").Logger(),
-		me:     me,
-		signer: signer,
-		state:  state,
-		client: client,
-		wait:   time.Second * 10,
+		log:               log.With().Str("module", "root_qc_voter").Logger(),
+		me:                me,
+		signer:            signer,
+		state:             state,
+		qcContractClients: contractClients,
```

Comment:
can we initialize `activeQCContractClient` to 0 here, just so it is explicit and the reader doesn't have to know about Go's initialization rules to reason about the behaviour. (same for Broker)